cmd/execution_config.go

```diff
@@ -73,7 +73,7 @@ func (exeConf *ExecutionConfig) SetupFlags(flags *pflag.FlagSet) {
 	flags.StringVarP(&exeConf.rpcConf.ListenAddr, "rpc-addr", "i", "localhost:9000", "the address the gRPC server listens on")
 	flags.UintVar(&exeConf.rpcConf.MaxMsgSize, "rpc-max-message-size", grpcutils.DefaultMaxMsgSize, "the maximum message size in bytes for messages sent or received over grpc")
 	flags.BoolVar(&exeConf.rpcConf.RpcMetricsEnabled, "rpc-metrics-enabled", false, "whether to enable the rpc metrics")
-	flags.StringVar(&exeConf.triedir, "triedir", datadir, "directory to store the execution State")
+	flags.StringVar(&exeConf.triedir, "triedir", filepath.Join(datadir, "trie"), "directory to store the execution State")
```

Comment:
the previous default value would not persist the trie data. 

Use `/data/trie` instead of `/trie` by default, so that the operator doesn't have to config this flag in order to persist the trie data